accesstoken/src/test/java/uk/gov/di/ipv/cri/common/api/handler/pact/utils/HandlerIntegrationTest.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.di.ipv.cri.common.api.handler.pact.utils;
+
+import java.lang.reflect.Field;
+import java.net.HttpCookie;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+public abstract class HandlerIntegrationTest<Q, S> {
```

Comment:
This can be converted into a static utility class. Generally we want to avoid abstract classes in Java since they're designed for refactoring. In the long run, abstract classes make it harder to refactor in the future.